src/main/resources/scripts/plant with hoe.sk

```diff
@@ -9,24 +9,15 @@ aliases:
 	# please note that changing this list won't affect which blocks actually break a cactus.
 	allowed next to cactus = 0, 6, 9, 10, 11, 27, 28, 31, 32, 37, 38, 39, 40, 50, 51, 55, 59, 65, 66, 75, 76, 77, 78, 83, 90, 93, 94, 104, 105, 106, 111, 115, 119
 
-on rightclick on grass, dirt or soil with any hoe:
-	block above is air
-	player has seeds
-	
-	set block above to crops
-	remove seeds from player
-	damage tool by 1
-	
-	block is not soil:
-		set block to soil
-		damage tool by 1 # damages the hoe a second time if the ground was not soil before
-
-# right click sand to plant a cactus
-on rightclick on sand with any hoe:
-	player has a cactus
-	block above is air
-	blocks north above, east above, south above and west above are allowed next to cactus
-	
-	damage tool by 1
-	set block above to cactus
-	remove a cactus from player
+on right click:

```

Comment:
Same thing as above, keeping triggers modularized is good.

Reply:
what do you mean? like `on right click of grass, dirt or soil:`?

Reply:
Yes. Latest release has them bugged. That is not a reason to change example scripts.

Reply:
alright, i'll change it back, I just didn't want it erroring on the latest release.